.changeset/few-rivers-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+feat(OnyxListbox): implement `check all` support + exclude disabled options from keyboard navigation
```

Comment:
Please also add a breaking change changeset for the removal of the checkAllLabel property